.travis.yml

```diff
@@ -47,6 +47,17 @@ jobs:
           env: DMD=2.085.* F=production
         - <<: *test-matrix-d2
           env: DMD=2.085.* F=devel
+        - <<: *test-matrix-d2
+          env: DIST=bionic DMD=2.078.3.s* F=production
```

Comment:
Adding explicit `DIST=xenial` to old rows would make it slightly clearer without any further context.

Reply:
right, I will add it to the old rows soon